### ISSUE TITLE
i'm very like this project, do you have any roadmap plans for it?

### ISSUE THREAD

**Issue by arden** [VolantMQ/volantmq#18]
i'm very like this project, do you have any roadmap plans for it? and have any plans to support cluster, websocket?

**Comment by troian** [comment]
I'll have roadmap next week. But generally it's hard to follow it as this project is a hobby until someone else start contributing "full time".
 - Websocket is easy to implement. So might be in near future.
 - Cluster is much more difficult.
If in brief I want to firstly port all of paho tests (including interoperability), then impelement permissions and do first release. Further investigate cluster, AMQP and possibly STOMP

**Comment by LeeWong** [comment]
Perhaps, this library can be useful for clustering: https://github.com/AsynkronIT/protoactor-go

**Comment by troian** [comment]
@arden websockets feature has been merged recently. You're welcome to check/comment and open issues.
